srcpkgs/yad/template

```diff
@@ -12,15 +12,9 @@ license="GPL-3.0-or-later"
 homepage="https://github.com/v1cont/yad"
 changelog="https://raw.githubusercontent.com/v1cont/yad/master/NEWS"
 distfiles="https://github.com/v1cont/yad/releases/download/v${version}/yad-${version}.tar.xz"
-checksum=5c9538b7f242de715249e9f7d30108c0706d23219b5b1bb85cfead6ae77abff3
+checksum=426f14156f1239f905d4ea4fd7fe59badaa7f6c524c1bc94ec0ef5682b07350f
 
 pre_configure() {
 	autoreconf -ivf
 	intltoolize
 }
-
-post_install() {
-	# Avoid conflict with disputils
-	mv "${DESTDIR}"/usr/bin/{,yad-}pfd
-	mv "${DESTDIR}"/usr/share/man/man1/{,yad-}pfd.1
-}
```

Comment:
Why are you removing this?

Reply:
My bad, I've read about removing pfd in changelog but apparently it's still there as a wrapper for yad-tools.